neqo-transport/src/recovery.rs

```diff
@@ -209,12 +227,152 @@ impl LossRecoverySpaces {
     }
 }
 
+#[derive(Debug)]
+struct CongestionControl {
```

Comment:
I know that this is small, but you should move it into its own file.

Reply:
#351